server/src/test/java/io/crate/action/sql/CursorTest.java

```diff
@@ -322,5 +432,33 @@ public void test_scrolling() throws Exception {
                 2
                 """
         );
+
+        assertThat(result).isNotCompleted();
+        cursor.close();
+        assertThat(result).isCompleted();
+    }
+
+    private static class SimpleRamAccounting implements RamAccounting {
```

Comment:
I think `new BlockBasedRamAccounting(bytes -> {}, 1)` would do the same.

Reply:
Thx!